tests/image-stream/utils.spec.js

```diff
@@ -26,44 +26,34 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
+    it('should resolve application/x-bzip2 for a bz2 archive', function() {
```

Comment:
Nope, quite the opposite – mocha knows how to handle promises and the `done()` in this case would stall the test on failure as unhandled exceptions from promises don't throw and don't terminate the process – which is why the promise is *returned* and no `done()` is being used.